seqr/fixtures/social_auth.json

```diff
@@ -17,7 +22,12 @@
         "user": 11,
         "provider": "google-oauth2",
         "uid": "test_user_manager@test.com",
-        "extra_data": "{\"expires\": 3599, \"auth_time\": 1603287741, \"token_type\": \"Bearer\", \"access_token\": \"ya29.EXAMPLE\"}",
+        "extra_data": {
```

Comment:
I confirmed that this aligns with whats actually in the prod database - `extra_data` is a jsonb field, not a string